2019-12-08 22:14:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "NEW[*]) "; SQL statement:
new) [42000-197]
2019-12-08 22:14:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" not found; SQL statement:
SELECT * FROM Players [42102-197]
2019-12-08 22:15:48 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Mikko/ot2019/Shakki/chessDatabase.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.mycompany.shakki.dao.LeaderboardDao.connectH2(LeaderboardDao.java:98)
	at com.mycompany.shakki.dao.LeaderboardDao.getPlayers(LeaderboardDao.java:41)
	at com.mycompany.shakki.gui.ShakkiUI.init(ShakkiUI.java:65)
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:841)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$154(LauncherImpl.java:182)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Mikko/ot2019/Shakki/chessDatabase.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Mikko/ot2019/Shakki/chessDatabase.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 24 more
2019-12-08 22:15:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "NEW[*]) "; SQL statement:
new) [42000-197]
2019-12-08 22:15:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" not found; SQL statement:
SELECT * FROM Players [42102-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
2019-12-08 22:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" already exists; SQL statement:
CREATE TABLE Players (name VARCHAR(255) PRIMARY KEY, wins INTEGER, losses INTEGER) [42101-197]
